build/charts/antrea/templates/webhooks/validating/crdvalidator.yaml

```diff
@@ -116,9 +116,9 @@ webhooks:
         namespace: {{ .Release.Namespace }}
         path: "/validate/externalippool"
     rules:
-      - operations: ["UPDATE"]
+      - operations: ["CREATE", "UPDATE"]
         apiGroups: ["crd.antrea.io"]
-        apiVersions: ["v1alpha2"]
+        apiVersions: ["v1alpha2", "v1beta1"]
```

Comment:
I missed this change before, should it be in a separate PR?

Reply:
I thought this is a bug and thought it couldn't work with listing v1beta1 here, however, it turns out working even without it because the `matchPolicy` defaults to `Equivalent`. So at least for now this could work with or without v1beta1.
I keep the change because I need to add "CREATE" anyway.

Reply:
Sounds good, no need for the separate PR then